plugin.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
+        id="edu.berkeley.eecs.emission.cordova.integritydetect"
+        version="1.0.0">
+
+  <name>IntegrityDetection</name>
+  <description>Integrity detection for road surface </description>
+  <author>Bingbing Huang (bing2013@cs.washington.edu)</author>
+  <license>Modified BSD license</license>
+
+  <engines>
+    <engine name="cordova" version=">=3.6.0"/>
+  </engines>
+
+  <platform name="android">
+    <config-file target="res/xml/config.xml" parent="/*">
+      <feature name="integrityDetect">
+        <param name="android-package" value="edu.berkeley.eecs.emission.cordova.integrityDetect.IntegrityNotificationReceiver"/>
+      </feature>
+    </config-file>
+
+    <source-file src="src/android/IntegrityNotificationReceiver.java" target-dir="src/edu/berkeley/eecs/emission/cordova/integrityDetect"/>
+    <source-file src="src/android/IntegrityDetector.java" target-dir="src/edu/berkeley/eecs/emission/cordova/integrityDetect"/>
+    <source-file src="src/android/SimpleMovementSensorEvent.java" target-dir="src/edu/berkeley/eecs/emission/cordova/integrityDetect"/>
+    <resource-file src="res/android/usercachekeys.xml" target="res/android/usercachekeys.xml"/>
+    <resource-file src="res/android/plugin_stats.xml" target="res/android/app_stats.xml"/>
```

Comment:
You should name these something different - otherwise when you add both plugins, these files will be overwritten in unpredictable ways. Also, I don't think you need `plugin_stats.xml`